examples/plugin/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.2.61'
+}
+
+apply plugin: 'application'
+
+applicationName = 'ort'
+mainClassName = 'com.here.ort.Main'
+
+repositories {
+    jcenter()
+
+    // Required for the gradle-tooling-api.
+    maven {
+        url 'https://repo.gradle.org/gradle/libs-releases-local/'
+    }
+}
+
+dependencies {
+    compile 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
+
+    // This example depends on the 'cli' module to re-use the command line interface.
```

Comment:
I'm not getting this. Where's the command line interface reused in the plugin code?